src/commands/protocols/common/submit-commit-command.js

```diff
@@ -5,15 +5,24 @@ import {
     COMMAND_TX_GAS_INCREASE_FACTORS,
 } from '../../../constants/constants.js';
 import Command from '../../command.js';
+import SendTransactionCommand from '../../common/send-transaction-command.js';
 
-class SubmitCommitCommand extends Command {
+class SubmitCommitCommand extends SendTransactionCommand {
     constructor(ctx) {
         super(ctx);
         this.commandExecutor = ctx.commandExecutor;
         this.blockchainModuleManager = ctx.blockchainModuleManager;
         this.operationIdService = ctx.operationIdService;
+        this.repositoryModuleManager = ctx.repositoryModuleManager;
 
         this.errorType = ERROR_TYPE.COMMIT_PROOF.SUBMIT_COMMIT_ERROR;
+
+        this.TX_START = OPERATION_ID_STATUS.COMMIT_PROOF.SUBMIT_COMMIT_SEND_TX_START;
```

Comment:
would make sense to be consistent here and use camelCase when defining these class variables, as it's what we did in all other classes